### DISCUSSION TITLE
GoPro Max - No conversion function to Lens Superview (fS)

### DISCUSSION THREAD

**Discussion by mariuszpasko** [gopro/labs#137] [General]
Hi.
As in the subject, the fS function does not work on the GoPro Max, I did not find a solution. Will this feature be available?

**Comment by dnewman-gpsw** [comment]
On Max it is not the same Superview at the HERO cameras.  fX is the command for MAX Superview.

**Comment by mariuszpasko** [comment]
I checked all options:

fN - does not work, switches to Wide
fL - works, switches to Linear
fW - works, switches to Wide
fS - does not work, switches to Wide
fX - doesn't work, no reaction

**Reply by dnewman-gpsw** [reply]
I will retest on the release Labs firmware, might be a bug.

**Reply by dnewman-gpsw** [reply]
Seems to be a bug in 2.0.71, hope to get an update out.
